pkg/canary/reader/reader.go

```diff
@@ -14,12 +13,12 @@ import (
 	"time"
 
 	"github.com/gorilla/websocket"
+	loghttp "github.com/grafana/loki/pkg/loghttp/legacy"
+	"github.com/grafana/loki/pkg/logproto"
+	json "github.com/json-iterator/go"
```

Comment:
Imports not sorted with `goimports -local github.com/grafana/loki`

Reply:
This applies for many other places in this PR as well, let's just run it on the entire project in a separate PR if there are too many places throughout the codebase :D

Reply:
> goimports -local github.com/grafana/loki

Where do we document this?

Reply:
We don‘t really do, but most Go projects shall accord to „Effective Go“ which recommends the following import structure:

```go
import (
  stdlib
  \n
  external libraries
  \n
  local packages
)

Reply:
I'm asking because I see different files with slightly different import order, and since I haven't seen this specified anywhere, my IDE was also set to simply run plain 'goimports' with no parameters.

We should agree on common settings and document it somewhere, preferably makefile/lint check.

Reply:
Yeah this should be catched by the linter.

Reply:
issue created.